README.md

```diff
@@ -123,3 +125,10 @@ An up object can be one of:
   - `met?` *[string, shell script]* the command to run to know if we are currently meeting the requirement
   - `unmeet` *[string, shell script]* the command to run to 'unmeet' the requirement during tear down
 
+## Configuration
+
+### Environment values
+
+* `OMNI_GIT` [path] The workspace directory where omni will clone and look for repositories. Defaults to `~/git` if it exists, or `$GOPATH` if it is defined.
+* `OMNIDIR` [path] The path to the omni repo. Defaults to searching under `$OMNI_GIT`.
+* `OMNI_ORG` [comma-delimited list of strings] Sets up implict trust for repositories within organizations. Organizations are assumed to be certain prefixes on the git origin. Example: `OMNI_ORG="git@github.com/xaf,github.com/xaf"`.
```

Comment:
Useful addition!

```suggestion
* `OMNI_GIT` *[path]* The workspace directory where omni will clone and look for repositories. Defaults to `~/git` if it exists, or `$GOPATH/src` if it is defined.
* `OMNIDIR` *[path]* The path to the omni repo. Defaults to searching under `$OMNI_GIT`.
* `OMNI_ORG` *[comma-delimited list of strings]* Sets up implicit trust for repositories within organizations. Organizations are assumed to be certain prefixes on the git origin. Example: `OMNI_ORG="git@github.com/XaF,github.com/XaF"`.
```

^ not sure how to formulate, but on top of implicit trust, organizations are to allow `omni clone` to close directly from repo name or org/repo if org is not specified. e.g. `omni clone omni` will clone `omni` from `github.com/XaF/omni` with those example organizations, while if `github.com/oefd` is the first organization, `omni` would be found there first!

Reply:
I tried to make it a bit more clear by breaking it down a bit more